src/main/java/Duke.java

```diff
@@ -1,10 +1,258 @@
+        } catch (ArrayIndexOutOfBoundsException e) {
+            System.out.println("Error: Invalid number!");
+        }
+    }
+
+    public static void addToDo(String description) {
+        list[inputCount] = new ToDo(description);
+        System.out.println("______________________________________________");
+        System.out.println("Got it. I've added this task:");
+        System.out.println("   " + list[inputCount].toString());
+        System.out.println("Now you have " + (inputCount + 1) + " tasks in the list.");
+        System.out.println("______________________________________________");
+        inputCount++;
+    }
+
+    public static void loadToDo(int i, String description) {
+        list[i] = new ToDo(description);
+        inputCount++;
+    }
+
+    public static void addDeadline(String desc) {
+        String[] splitDesc = desc.split("/by");
+        list[inputCount] = new Deadline(splitDesc[0], splitDesc[1]);
+        System.out.println("______________________________________________");
+        System.out.println("Got it. I've added this task:");
+        System.out.println("   " + list[inputCount].toString());
+        System.out.println("Now you have " + (inputCount + 1) + " tasks in the list.");
+        System.out.println("______________________________________________");
+        inputCount++;
+    }
+
+    public static void loadDeadline(int i,String desc) {
+        String[] splitDesc = desc.split("/by");
+        list[i] = new Deadline(splitDesc[0], splitDesc[1]);
+        inputCount++;
+    }
+
+    public static void addEvent(String event) {
+        String[] splitEvent = event.split("/at");
+        list[inputCount] = new Event(splitEvent[0],splitEvent[1]);
+        System.out.println("______________________________________________");
+        System.out.println("Got it. I've added this task:");
+        System.out.println(list[inputCount].toString());
+        System.out.println("Now you have " + (inputCount + 1) + " tasks in the list.");
+        System.out.println("______________________________________________");
+        inputCount++;
+    }
+
+    public static void loadEvent(int i, String event) {
+        String[] splitEvent = event.split("/at");
+        list[i] = new Event(splitEvent[0],splitEvent[1]);
+        inputCount++;
+    }
+
+    public static String toSave(Task t) {
+        return(t.getType() + " > " + t.isDone() + " > " + t.toStringToSave());
+    }
+
+    private static void loadFileContents(String filePath) throws FileNotFoundException {
+        File f = new File(filePath);
+        Scanner sc = new Scanner(f);
+        if(!sc.hasNext()) {
+            inputCount = 0;
+        } else {
+            int inputCount = Integer.parseInt(sc.nextLine());
+            for (int i = 0; i < inputCount; i++) {
+                String taskLine = sc.nextLine();
+                String[] taskDetails = taskLine.split(" > ");
+                if (taskDetails[0].equals("E")) {
```

Comment:
how about change this to a switch statement, then do list[i].isDone = Boolean.parseBoolean(taskDetails[1]); at the end of the switch since all 3 lines are doing the same thing (line 144, 148, 152)